mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule OMG.Umbrella.MixProject do
   use Mix.Project
 
-  def umbrella_version, do: "0.2.0"
+  def umbrella_version, do: "#{String.trim(File.read!("VERSION"))}"
```

Comment:
not needed anymore I think, it was only there to be able to modify all `version` entries in one place, which is done via `VERSION` now

Reply:
It's now being used in https://github.com/omisego/elixir-omg/blob/master/Makefile#L116
and https://github.com/omisego/elixir-omg/blob/master/Makefile#L123 
so that releases can be built :)